tdrs-backend/tdpservice/users/migrations/0015_replace_data_analyst_role.py

```diff
@@ -0,0 +1,21 @@
+
+# Generated by Django 3.1.4 on 2021-01-08 17:41
+
```

Comment:
This was the only chunk I actually had to write anything. The rest of the changes were done through 3 different search and replaces over python files, the term showed up with 3 different spacing patterns.